include/mlir/Transforms/LoopUtils.h

```diff
@@ -24,11 +24,12 @@
 #ifndef MLIR_TRANSFORMS_LOOP_UTILS_H
 #define MLIR_TRANSFORMS_LOOP_UTILS_H
 
+#include "mlir/IR/AffineMap.h"
```

Comment:
Why is this include necessary? Prefer forward declarations whenever possible.

Reply:
Actually neither the include nor the forward declaration was necessary here. Thanks .